README.md

```diff
@@ -81,7 +81,7 @@ client.login('token');
 * [Website](https://discord.js.org/) ([source](https://github.com/discordjs/website))
 * [Documentation](https://discord.js.org/#/docs/main/master/general/welcome)
 * [Guide](https://discordjs.guide/) ([source](https://github.com/discordjs/guide)) - this is still for stable  
-  See also the WIP [Update Guide](https://github.com/discordjs/guide/blob/v12-changes/guide/additional-info/changes-in-v12.md) also including updated and removed items in the library.
+  See also the WIP [Update Guide](https://github.com/discordjs/guide/blob/master/guide/additional-info/changes-in-v12.md) also including updated and removed items in the library.
```

Comment:
Might as well remove the `WIP` too, i also dont see why you dont just link to the actual guide which looks better https://discordjs.guide/additional-info/changes-in-v12.html

Reply:
Not only looks better, but actually works. All links like those:
![image](https://user-images.githubusercontent.com/31581159/73146857-d9c3cc00-40b4-11ea-944d-de9c3b915067.png)
do not link to github repo, but are internal links for guide's website itself, meaning that if you click on them on github, they don't work at all.

Reply:
True, forgot about that too 👍